django/cantusdb_project/main_app/models/cantus_identifier.py

```diff
@@ -0,0 +1,18 @@
+from main_app.models import BaseModel
+from django.db import models
+
+
+class CantusIdentifier(BaseModel):
+    cantus_id = models.CharField(blank=False, null=False, max_length=255)
```

Comment:
- `blank` and `null` are False by default, so you don't need to specify them. 
- 255 chars for an identifier is pretty long, no?

Reply:
> `blank` and `null` are False by default, so you don't need to specify them.

This is a good point. We've got this boilerplate in a bunch of places in our models, iirc - might be worth cleaning up at some point.

> 255 chars for an identifier is pretty long, no?

This is a good point - @lucasmarchd01, 15=(2**4)-1 might be a better number?